sym/ca3e6c8e-5035-4685-a043-0f73c891df55/symbol.lp

```diff
@@ -0,0 +1,53 @@
+(librepcb_symbol ca3e6c8e-5035-4685-a043-0f73c891df55
+ (name "Buffer With Inverted Enable")
+ (description "Quad buffer/line driver; 3-state; Inverted Enable")
+ (keywords "74HCT145, 74HC145")
+ (author "Shibin Das")
+ (version "0.1")
+ (created 2019-04-14T14:10:14Z)
+ (deprecated false)
+ (category 56f01b1d-0723-4c33-88f7-9968cf96b50f)
+ (category a4c3b94b-5321-4adb-8a77-d3b53f178cee)
+ (pin e36b3fb3-5686-4a1d-89d1-7ab2dfcaa03b (name "OE")
+  (position -2.54 -5.08) (rotation 0.0) (length 2.54)
```

Comment:
I would suggest to move this pin 2.54mm to the right, then you don't force to connect netlines from the left. This way it can be connected from the left, right or bottom.

```suggestion
  (position 0.0 -5.08) (rotation 90.0) (length 1.27)
```